pom.xml

```diff
@@ -580,6 +580,8 @@
 
       <properties>
         <docker.context>incubator</docker.context>
+        <docker.platforms>linux/x86_64,linux/arm64</docker.platforms>
+        <docker.tagAlt>edge</docker.tagAlt>
```

Comment:
This naming can be updated, I just picked what I normally see in other docker repos. `edge` is a more common tag for the head of develop vs `developer-SNAPSHOT` which is specific to java.